From 88a701106df6c56548e10fed41511e4560031f92 Mon Sep 17 00:00:00 2001
From: Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
Date: Wed, 25 Mar 2015 10:28:51 -0300
Subject: [PATCH] Allow building from the source code directory

Remove the code that prevented building from the source code directory.
Regenerate configure.
---
 config.h.in  |    3 +
 configure    | 3670 +++++++++++++++++++++++++---------------------------------
 configure.ac |    5 -
 3 files changed, 1604 insertions(+), 2074 deletions(-)

diff --git a/config.h.in b/config.h.in
index 3740229..6f1f697 100644
--- a/config.h.in
+++ b/config.h.in
@@ -55,6 +55,9 @@
 /* Define to the one symbol short name of this package. */
 #undef PACKAGE_TARNAME
 
+/* Define to the home page for this package. */
+#undef PACKAGE_URL
+
 /* Define to the version of this package. */
 #undef PACKAGE_VERSION
 
diff --git a/configure.ac b/configure.ac
index 3e5272d..661589a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -6,11 +6,6 @@ AC_INIT(auxv, 1.0.0, rsa@us.ibm.com)
 AC_CONFIG_SRCDIR([lsauxv.c])
 AC_CONFIG_HEADER([config.h])
 
-# We want to prevent building/configuration in the source directory.
-if test "`cd $srcdir; /bin/pwd`" = "`/bin/pwd`"; then
-	AC_MSG_ERROR([you must configure in a separate build directory])
-fi
-
 AC_ARG_ENABLE([shared],
 	      AS_HELP_STRING([--enable-shared],
 			     [build shared library @<:@default@:>@]),
